src/cascadia/TerminalSettingsModel/CascadiaSettingsSerialization.cpp

```diff
@@ -278,9 +363,11 @@ void SettingsLoader::FindFragmentsAndMergeIntoUserSettings()
                 const auto filename = fragmentExtFolder.path().filename();
                 const auto& source = filename.native();
 
-                if (!_ignoredNamespaces.contains(std::wstring_view{ source }) && fragmentExtFolder.is_directory())
+                if (fragmentExtFolder.is_directory())
                 {
-                    parseAndLayerFragmentFiles(fragmentExtFolder.path(), winrt::hstring{ source });
+                    parseAndLayerFragmentFiles(fragmentExtFolder.path(),
+                                               winrt::hstring{ source },
+                                               rfid == FOLDERID_LocalAppData ? FragmentScope::User : FragmentScope::Machine); // scope

```

Comment:
fwiw you could structure this as follows:

```c++
auto scope = FragmentScope::User;
for(const auto& rfid : std::array{ RFID, RFID }) {
    for (...) {
        something(scope);
    }
    // Since the outer loop only has two entries, set the scope for the next run
    scope = FragmentScope::Machine;
}
```

that lets you get rid of one pesky ternary _and_ a comparison against one of the array entries; TBD which is more fragile

Reply:
Personally, I find it more fragile to set the scope once. Makes me feel a little icky haha.